### ISSUE TITLE
Hanging after compilation?

### ISSUE THREAD

**Issue by adamholdenyall** [monounity/karma-typescript#81]
After running karma-typescript all day, I've started running into an issue in the last hour when performing a ```karma start```. I get the "Compiled X files in Y ms message... but I never see the bundling message and the node process in the task manager starts eating up CPU and memory until I kill it with the task manager.

I even pulled down the master.zip and tried running the angular2 example and I'm getting the same behavior. I'm not sure how to get more verbose information about what is going on, or if there are caches that I can try to clear... curious if anyone else is experiencing this right now, possibly due to an incompatible new package getting published?

**Comment by erikbarke** [comment]
Hey @adamholdenyall, if you enable debug logging in the Karma conf, `logLevel: config.LOG_DEBUG`, how far does the plugin get?

**Comment by adamholdenyall** [comment]
I haven't been able to reproduce this in the morning. I did figure out how to run it with Debug logging one time last night, but it didn't print anything after the "Compiled..." message. 

I ended up switching my npm setup to the global repository at home last night, and when I reinstalled it worked fine. I'd like to chalk it up to issues with my office network/npm setup, but when I came in this morning and ran karma on the build server that was experiencing the same issue, it ran fine. 

In short, I guess no one else is running into this, and unless it happens again and I can get you better logging, it was just an inconvenient fluke.

**Comment by erikbarke** [comment]
Ok, closing this issue!
